scripts/deploy/deploy-production.ts

```diff
@@ -1,105 +1,87 @@
 import prompts from "prompts";
 import { tagUtils } from "./utils/git/tag";
-import { green, underline } from "../utils/colors";
 import { isRunningOnCI } from "./utils/environment";
 import { getCommitMessages, getCurrentlyDeployedCommit } from "./utils/git";
 import { runDeploy } from "./utils/deploy";
 
-const { createNewTag, deleteTag, getLatestTag, pushTags } = tagUtils;
+const { createTagAndPush, deleteTag, getLatestTag, pushTags } = tagUtils;
 /* Deploy by pushing a git tag, to be picked up and built by Evergreen, and deployed to S3. */
 const evergreenDeploy = async () => {
-  const currentlyDeployedCommit = getCurrentlyDeployedCommit();
-  console.log(`Currently Deployed Commit: ${currentlyDeployedCommit}`);
+  try {
+    const currentlyDeployedCommit = getCurrentlyDeployedCommit();
+    console.log(`Currently Deployed Commit: ${currentlyDeployedCommit}`);
 
-  const commitMessages = getCommitMessages(currentlyDeployedCommit);
+    const commitMessages = getCommitMessages(currentlyDeployedCommit);
 
-  // If there are no commit messages, ask the user if they want to delete and re-push the latest tag, thereby forcing a deploy with no new commits.
-  if (commitMessages.length === 0) {
-    const latestTag = getLatestTag();
-    const response = await prompts({
-      type: "confirm",
-      name: "value",
-      message: `No new commits. Do you want to trigger a deploy on the most recent existing tag? (${latestTag})`,
-      initial: false,
-    });
+    // If there are no commit messages, ask the user if they want to delete and re-push the latest tag, thereby forcing a deploy with no new commits.
+    if (commitMessages.length === 0) {
+      const latestTag = getLatestTag();
+      const { value: shouldForceDeploy } = await prompts({
+        type: "confirm",
+        name: "value",
+        message: `No new commits. Do you want to trigger a deploy on the most recent existing tag? (${latestTag})`,
+        initial: false,
+      });
```

Comment:
We ran into an issue last week when the deployer just hit enter and breezed through this message. Could we maybe have it default to false so that a user needs to be a little more careful to go through with this action?